src/axolotl/utils/config.py

```diff
@@ -339,6 +339,11 @@ def validate_config(cfg):
             "push_to_hub_model_id is deprecated. Please use hub_model_id instead."
         )
 
+    if cfg.hub_model_id and not (cfg.save_steps or cfg.saves_per_epoch):
```

Comment:
Should this also check for `cfg.save_strategy`?

Reply:
Yeah your right @NanoCode012 . The test should check for save_strategy, and the comments should be something like: 

`hub_model_id is set without any models being saved. To save a model, set save_strategy to steps or leave empty. ` Dose that sound right? If so I can create a pr and update.

Reply:
I am a bit confused both save_on_epochs and save_steps needs save_strategy to be steps see: https://github.com/OpenAccess-AI-Collective/axolotl/blob/5a5d47458d9aaf7ead798d15291ba3d9bef785c5/src/axolotl/utils/config.py#L426 
https://github.com/OpenAccess-AI-Collective/axolotl/blob/5a5d47458d9aaf7ead798d15291ba3d9bef785c5/src/axolotl/utils/config.py#L442

However https://github.com/OpenAccess-AI-Collective/axolotl/blob/5a5d47458d9aaf7ead798d15291ba3d9bef785c5/README.md?plain=1#L777 says nothing about that, it only says set to no to skip. Should we also maybe update the comment to be somthing like set to 'steps' to save checkpoints?

Reply:
`save_strategy` and `save_steps` are default HF Trainer Arguments. `save_on_epochs` is just a handy utility by axolotl that auto computes the `save_steps` based on total steps, hence needing steps.

This is the doc for `save_strategy`: https://huggingface.co/docs/transformers/v4.37.2/en/main_classes/trainer#transformers.TrainingArguments.save_strategy